manifest.php

```diff
@@ -24,10 +24,10 @@
     'label'       => 'Test Booklets',
     'description' => 'An extension for TAO to create test booklets (publishable in MS-Word and PDF along with Answer Sheets)',
     'license'     => 'GPL-2.0',
-    'version'     => '2.0.0',
+    'version'     => '2.1.0',
```

Comment:
This is breaking change any extended class of the `AbstractBookletController` will not work as before.